data/json/snippets/children_notes.json

```diff
@@ -137,5 +137,13 @@
         "text": "\"In case of emergency fill out this form so that you may present it to emergency personnel for identification purposes.  Keep this form with your emergency supplies.  If You need to request new copies of this form, speak with the school safety office.  Remember to keep calm, and stay in one location if possible, this will help emergency personnel to find you.  If this form is found please contact Oak Grove Elementary School.  Name: Aiden.  Parent or Guardian: Mommy Daddy.  Phone number where you can be reached: I don't remember I think it has a 5 in it.  Teacher: Miss daisy.  Current location: I dont no I’m scared, Im hear all alone.  Medical Conditions i.e diabetes: I dont no what that is.  Additional Information: Why is everyone acting so weird?!\""
       }
     ]
+  },
```

Comment:
You've added these below outside of the snippet object above.  Get rid of the `]` on line 139.